### DISCUSSION TITLE
Support for Swaggerdocs

### DISCUSSION THREAD

**Discussion by mdeliw** [squidfunk/mkdocs-material#2374] [General]
Checking with the contributors on how are users integrating mkdocs with swagger content? I have checked foliant, slate, widdershins etc and I am yet to find something that integrates with multiple swagger docs from several remote projects.

**Comment by squidfunk** [comment]
Material for MkDocs does only render Markdown - nothing more. If you can output Markdown from Swagger, you could throw it at this project and it should look great.

**Comment by jromal** [comment]
It is an old discussion that I found looking for "swagger" support.
There is an mkdocs plugin https://github.com/bharel/mkdocs-render-swagger-plugin that works very well according to my initial tests.
In case someone does search for it.
